threaddb/threaddb.go

```diff
@@ -0,0 +1,146 @@
+package threaddb
```

Comment:
New package here that both `buckets` and `mail` can use for creating, saving, deleting, etc., threaddb instances.